source/spark_sql_migrations/spark_sql_migrations/current_state/create_current_state.py

```diff
@@ -0,0 +1,43 @@
+﻿import spark_sql_migrations.infrastructure.sql_file_executor as sql_file_executor
+from importlib.resources import contents
+from dependency_injector.wiring import Provide, inject
+from spark_sql_migrations.container import SparkSqlMigrationsContainer
+from spark_sql_migrations.models.configuration import Configuration
+
+
+def create_all_tables() -> None:
+    _create_all_tables()
+
+
+@inject
+def _create_all_tables(
+    config: Configuration = Provide[SparkSqlMigrationsContainer.config]
+) -> None:
+    """Executes schema and table scripts to create all tables"""
+    print("Creating all tables")
+    schema_scripts = _get_schema_scripts()
+    table_scripts = _get_table_scripts()
+
+    for script in schema_scripts:
+        sql_file_executor.execute(script, config.current_state_schemas_folder_path)
+
+    for script in table_scripts:
+        sql_file_executor.execute(script, config.current_state_tables_folder_path)
+
+    print("Successfully created all tables")
+
+
+@inject
+def _get_schema_scripts(
+        config: Configuration = Provide[SparkSqlMigrationsContainer.config]
+) -> list[str]:
+    migration_files = list(contents(config.current_state_schemas_folder_path))
```

Comment:
is list necessary? Isn't contens iterable?

Reply:
it is. i removed list